h5py/h5t.pyx

```diff
@@ -14,6 +14,7 @@
     subclasses which represent things like integer/float/compound identifiers.
     The majority of the H5T API is presented as methods on these identifiers.
 """
+from __future__ import print_function
```

Comment:
I'm presuming this is left over from debugging?

Reply:
Yeah, sorry. Pushing its removal.